pos_product_multi_price/models/product.py

```diff
@@ -0,0 +1,24 @@
+import json
+from odoo import models, fields, api
+
+
+class Product(models.Model):
+    _inherit = 'product.product'
+
+    # technical field used in POS frontend
+    price_ids_json = fields.Char(
+        "Multi price data dict", readonly=True,
+        compute="_compute_price_ids_json")
```

Comment:
```suggestion
    price_ids_json = fields.Char(
        "Multi price data dict",
        help="Technical field used in POS frontend",
        compute="_compute_price_ids_json",
    )
```

- computed fields are already readonly by default.
- you can move the comment to the help attribute, it's more helpful this way